src/main/resources/application.properties

```diff
@@ -25,9 +25,9 @@ spring.liquibase.contexts=p
 # VIEW -----------------------------------------------------------------------------------------------------------------
 spring.mvc.view.prefix=/WEB-INF/jsp/
 spring.mvc.view.suffix=.jsp
-#server.jsp-servlet.init-parameters.development=false
+server.servlet.jsp.init-parameters.development=false
```

Comment:
Würde ich hier nicht explizit konfigurieren, da schon in [Jsp.java](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/servlet/server/Jsp.java#L45) dies für uns gesetzt wird

Reply:
hab den Commit dazu rausgenommen